src/main/java/bio/overture/ego/config/SecureServerConfig.java

```diff
@@ -141,7 +143,9 @@ protected void configure(HttpSecurity http) throws Exception {
               "/configuration/ui",
               "/configuration/**",
               "/v2/api**",
-              "/webjars/**")
+              "/webjars/**",
+              "/oauth/token/public_key",
+              "/oauth/ego-token")
```

Comment:
ego-token actually belongs back where it was. This is because `/oauth/ego-token` is the final piece of the OAuth2 redirecting, where the user agent will exchange a session cookie for a user JWT.